frontend/appflowy_flutter/lib/plugins/ai_chat/presentation/chat_input/select_sources_bottom_sheet.dart

```diff
@@ -205,32 +204,27 @@ class _MobileSelectSourcesSheetBodyState
         ),
         BlocBuilder<ChatSettingsCubit, ChatSettingsState>(
           builder: (context, state) {
+            final sources = state.visibleSources
+                .where((e) => e.ignoreStatus != IgnoreViewType.hide)
+                .toList();
             return SliverList(
               delegate: SliverChildBuilderDelegate(
-                childCount: state.visibleSources.length,
+                childCount: sources.length,
                 (context, index) {
                   return ChatSourceTreeItem(
```

Comment:
Above this line you can do `final source = sources[index];` instead of accessing it multiple times.

Reply:
Thanks, fixed!